eng/tox/tox.ini

```diff
@@ -315,7 +315,7 @@ deps =
   {[base]deps}
 commands =
     # install API stub generator
-    {envbindir}/python -m pip install "git+https://github.com/azure/azure-sdk-tools.git@api-stub-generator_v0.2.10#subdirectory=packages/python-packages/api-stub-generator&egg=api-stub-generator"
+    {envbindir}/python -m pip install api-stub-generator --index-url="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/"
```

Comment:
I am worried about leaving this unpinned. Hotfix release, etc, I don't think `always latest` will be a good experience. 

The retargeted index url definitely should stick around though. Thanks!

Reply:
Per our Teams discussion, we'll keep this latest for now.